js/app.jsx

```diff
@@ -159,6 +187,106 @@ var SearchableRemoteListMixin = {
 
 }
 
+var ListToolbar = React.createClass({
```

Comment:
Currently I handle using `if...else`, maybe separate them into different class will be a better way.